Assignment6/app.py

```
@@ -0,0 +1,207 @@
+def not_found(error):
+    return jsonify({'error': 'Not found'}), 404
+
+@app.errorhandler(500)
+def internal_error(error):
+    return jsonify({'error': 'Internal server error'}), 500
+
+if __name__ == '__main__':
+    # Initialize database
+    init_db()
+    # Run the application
+    app.run(host='0.0.0.0', port=8000, debug=True)
```

Ruff (0.14.7)

Possible binding to all interfaces

(S104)